frontend/app/src/components/ToolbarActions/ToolbarActions.test.tsx

```diff
@@ -36,31 +37,26 @@ describe("ActionButton", () => {
   })
 
   it("renders without crashing and matches snapshot", () => {
```

Comment:
Nit: Prefer just calling this test `renders without crashing` & removing snapshot from it since don't think it adds value in this case